src/coreclr/jit/CMakeLists.txt

```diff
@@ -363,6 +363,15 @@ convert_to_absolute_path(JIT_ARM_SOURCES ${JIT_ARM_SOURCES})
 convert_to_absolute_path(JIT_I386_SOURCES ${JIT_I386_SOURCES})
 convert_to_absolute_path(JIT_ARM64_SOURCES ${JIT_ARM64_SOURCES})
 
+# Set natvis file
+
+set(JIT_NATVIS_SOURCE
+  clrjit.natvis
+)
+
+convert_to_absolute_path(JIT_NATVIS_SOURCE ${JIT_NATVIS_SOURCE})
+add_linker_flag("/NATVIS:${JIT_NATVIS_SOURCE}")
```

Comment:
All this natvis stuff is Windows-only

Reply:
Took a stab at it. @jkoritzinsky , let me know if there is anything that is not correct.